cmd/runtimetest/main.go

```diff
@@ -204,6 +204,39 @@ func validateSysctls(spec *specs.LinuxSpec, rspec *specs.LinuxRuntimeSpec) error
 	return nil
 }
 
+func validateMount(spec *specs.LinuxSpec, rspec *specs.LinuxRuntimeSpec) error {
+	fmt.Println("validating mount")
+	//read the /proc/mount file and covert to map[path]spec.mount
```

Comment:
I think this approach may be too simplistic.

* It doesn't handle the required mounts from opencontainers/specs#164.
* By the time you're inside the container checking the host-side source is difficult (although a Btrfs pivot-root entry has `subvol=${HOST_PATH_TO_PIVOT_ROOT}`, so maybe it's not impossible).

Instead of writing something that generically validates *any* bundle's mounts, it's probably easier and effective enough to write a few bundles that make specific mounts and then check for those after launching the bundle (e.g. does the container's `/file` match the bundle's `roofs/file`?  Is `/` read-only or not?  …).